FEATURES.md

```diff
@@ -0,0 +1,97 @@
+
+# Feature proposal process:
+
+The feature proposal is an issue filed on a repo in OpenSearch which outlines the statement of the problem (technical design is not needed).The issue doesn’t have to contain implementation or design details but it's recommended to include ideas of possible fixes. The primary goal is to brainstorm the solutions along with the community beginning from this feature proposal phase. The issue is tagged “[feature-proposal]” and each repo should have their own template or a fallback on a standard feature proposal template. The maintainers will triage the feature proposal and ask follow-up questions as needed. The maintainer’s job is to determine if the proposal meets the end-result criteria:
```

Comment:
Thanks @samuel-oci, I really like a lot of this and like the idea of committing it into a doc here! I also agree with you as you say below that today we have an informal convention of RFCs that often land with too much detail, which circumvents the "brainstorm solutions" phase with the community. My only issue here is that the feature proposal phrase would seem to imply that you've already got a solution in mind, and that the feature proposal templates actually require that you fill out a box that says "describe the solution you'd like". How can we ensure that this process doesn't discourage folks from opening issues when all they have is a problem statement but no clear idea of how to craft a feature to solve it?

Reply:
+1 I think we can add context on it being okay to create an issue before you have all the details and add more details iteratively.

Reply:
Hi @andrross @elfisher I'm with you on that. Right now the wording says that `it's recommended to include ideas of a possible fix`. If you guys would like we can change the wording here to something that better reflect the intent.